test/utils/nodeUtils.test.ts

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { inferPackageName, inferCommand, InspectMode, NodePackage } from "../../extension.bundle";
+
+ suite('utils/nodeUtils', () => {
+    suite('inferPackageName', () => {
```

Comment:
Maybe add a UT for the above case of a defined, empty package name?

Reply:
Added.